packages/mui-lab/src/Masonry/Masonry.test.js

```diff
@@ -112,9 +113,14 @@ describe('<Masonry />', () => {
       if (/jsdom/.test(window.navigator.userAgent)) {
         this.skip();
       }
-      expect(() => render(<Masonry columns={3} spacing={1} />)).toErrorDev(
-        'Warning: Failed prop type: The prop `children` is marked as required in `ForwardRef(Masonry)`, but its value is `undefined`.',
-      );
+
+      // React 19 removed prop types support
+      if (reactMajor < 19) {
```

Comment:
Let's just skip if the React version is >= 19, similar to the other tests.

Reply:
Doing so would skip the main part of this test (line 124), which checks "should not throw type error when children are empty"

Reply:
It seems like we can remove this `expect`:

```
expect(() => render(<Masonry columns={3} spacing={1} />)).toErrorDev(
  'Warning: Failed prop type: The prop `children` is marked as required in `ForwardRef(Masonry)`, but its value is `undefined`.',
);
```

Because it's already being tested in the previous test.

Reply:
Yeah agree.

Reply:
Updated

Reply:
I need to bring it back https://app.circleci.com/pipelines/github/mui/material-ui/135494/workflows/14045e23-4a31-4dc0-9422-bc8e9ffe3620/jobs/730406?invite=true#step-111-132

Reply:
I final brought back the check, otherwise the test fails in React < 19